websites/V/Vizjer/presence.ts

```diff
@@ -0,0 +1,187 @@
+			presenceData.smallImageKey = Assets.Viewing;
+			break;
+		case "/twoje-materialy":
+			presenceData.details = "Przegląda swoje materiały...";
+			presenceData.smallImageKey = Assets.Viewing;
+			break;
+		case movies:
+			presenceData.details = "Przegląda filmy online...";
+			if (movies.includes("sort:date")) {
+				presenceData.details =
+					"Przegląda ostatnio zaaktualizowane filmy online...";
+			}
+			if (movies.includes("sort:newlink"))
+				presenceData.details = "Przegląda ostatnio dodane filmy...";
+			if (movies.includes("sort:vote"))
+				presenceData.details = "Przegląda filmy z największą liczbą głosów...";
+			if (movies.includes("sort:premiere"))
+				presenceData.details = "Przegląda ostatnie premiery filmów...";
+			if (movies.includes("sort:view"))
+				presenceData.details = "Przegląda filmy z największą liczbą odsłon...";
+			if (movies.includes("sort:rate"))
+				presenceData.details = "Przegląda filmy z największą liczbą ocen...";
+			presenceData.smallImageKey = Assets.Search;
+			presenceData.state = search
+				? `Strona ${search.split("=")[1]}`
+				: "Strona 1";
+			break;
+		case series:
+			presenceData.details = "Przegląda seriale online...";
+			if (series.includes("sort:newest") || series.includes("sort:date"))
+				presenceData.details = "Przegląda najnowsze seriale online...";
+			if (series.includes("sort:year"))
+				presenceData.details = "Przegląda ostatnie premiery seriali...";
+			if (series.includes("sort:vote")) {
+				presenceData.details =
+					"Przegląda seriale z największą liczbą głosów...";
+			}
+			if (series.includes("sort:view")) {
+				presenceData.details =
+					"Przegląda seriale z największą liczbą odsłon...";
+			}
+			if (series.includes("sort:rate"))
+				presenceData.details = "Przegląda seriale z największą liczbą ocen...";
+			presenceData.smallImageKey = Assets.Search;
+			presenceData.state = search
+				? `Strona ${search.split("=")[1]}`
+				: "Strona 1";
+			break;
+		case "/dla-dzieci/":
+			presenceData.details = "Przegląda filmy i seriale dla dzieci...";
+			presenceData.state = search
+				? `Strona ${search.split("=")[1]}`
+				: "Strona 1";
+			presenceData.smallImageKey = Assets.Search;
+			break;
+		case "/premium":
+			presenceData.details = "Przegląda zakładkę premium...";
+			presenceData.smallImageKey = Assets.Viewing;
+			break;
+		case "/ranking":
+			presenceData.details = "Przegląda ranking...";
+			presenceData.smallImageKey = Assets.Viewing;
+			break;
+		case pathname.startsWith("/film/") ? pathname : null:
+			for (const info of document.querySelectorAll("div.info>ul")) {
+				if (info.children[0].textContent === "Rok: ") {
+					presenceData.details = `${
+						document.querySelector("h2>span").textContent.split("/")[0]
+					} (${info.children[1].textContent})`;
+				}
+			}
+			// presenceData.largeImageKey = document.querySelector("div#single-poster>img").getAttribute("src");
+			presenceData.buttons = [{ label: "Oglądaj", url: document.URL }];
```

Comment:
Use `document.href` rather than `document.URL` (multiple locations)

Reply:
![image](https://github-production-user-asset-6210df.s3.amazonaws.com/42370647/279045621-6d8961ed-87e1-40c3-999c-62865db0df12.png)

Reply:
ah you prob mean document.location.href